recipe/meta.yaml

```diff
@@ -25,44 +24,44 @@ requirements:
     - python
     - cymem 2.0.6
     - cython 0.29.32
-    - cython-blis 0.7.7
     - murmurhash 1.0.7
     - numpy {{ numpy }}
     - pathy 0.10.1
     - pip
     - preshed 3.0.6
     - setuptools
-    - thinc 8.0.15
+    - thinc 8.1.10
     - wheel
   run:
     - python
     # Spacy libraries
-    - spacy-legacy >=3.0.9,<3.1.0
+    - spacy-legacy >=3.0.11,<3.1.0
     - murmurhash >=0.28.0,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
-    - thinc >=8.0.14,<8.1.0
-    - cython-blis >=0.4.0,<0.8.0
-    - wasabi >=0.9.1,<1.1.0
+    - thinc >=8.1.8,<8.2.0
+    - wasabi >=0.9.1,<1.2.0
     - srsly >=2.4.3,<3.0.0
     - catalogue >=2.0.6,<2.1.0
     - spacy-loggers >=1.0.0,<2.0.0
-    # Exclude typer with unpinned click
-    - typer >=0.3.0,<0.5.0
+    # Exclude typer with unpinned click (>=0.3.2)
+    - typer >=0.3.2,<0.5.0
     # Exclude newer click due to typer issues
     - click <8.1.0
```

Comment:
Is this still valid? All typer versions we provide have an upper bound set for click.

Reply:
You are right. We can drop `click`, it was removed by the maintainers.